src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineBeginTreeLastVariableResourcesStatementEnd1.java

```diff
@@ -9,7 +9,6 @@
 
 import java.io.*;
 
-// ok
```

Comment:
Removal of this line changed the violation line.
You will need to change the violation lines of expected string array in the respective check.
Or you can simply add a blank line where the '// ok' comment has been removed.

Reply:
Opted for the second solution.